docs/johannes_mueller/yolo_from_omero/train_yolo.ipynb

```diff
@@ -0,0 +1,705 @@
+    }
+   ],
+   "source": [
+    "for obj_class, box in zip(prediction[0].boxes.cls, prediction[0].boxes.xywh):\n",
+    "    print(f'Object class: {int(obj_class)}, Box: {box.detach().cpu().numpy()}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Moving the data back to OMERO\n",
+    "\n",
+    "To come full circle, here's a quick glance on how to move the images along with the predictions back to OMERO. Luckily for us, ezomero makes this relatively straightforward:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "swapped_object_classes = {value: key for key, value in object_classes.items()}\n",
+    "\n",
+    "rectangles = []\n",
+    "for obj_class, box in zip(prediction[0].boxes.cls, prediction[0].boxes.xywh):\n",
+    "    box_numpy = box.detach().cpu().numpy()\n",
+    "    x = box_numpy[0] - box_numpy[2] / 2\n",
+    "    y = box_numpy[1] - box_numpy[3] / 2\n",
+    "    width = box_numpy[2]\n",
+    "    heigth = box_numpy[3]\n",
+    "    \n",
+    "    rectangles.append(ezomero.rois.Rectangle(x, y, width, heigth, label=swapped_object_classes[int(obj_class)]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lastly, we could upload this image and the corresponding predictions to a new dataset in OMERO. Note that the image has to be converted in 5d (in our case, we set the format to `tczyx`) to be compatible with OMERO.\n",
+    "\n",
+    "```{note}\n",
+    " The steps up to this point can take a bit of time so you may have to reconnect to the omero server.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_to_upload = io.imread(test_image)\n",
+    "image_id = ezomero.post_image(\n",
+    "    conn, image=image_to_upload[None, None, None, :], dim_order='tczyx',\n",
+    "    dataset_id=260, image_name='example_prediction'\n",
+    "    )\n",
+    "\n",
+    "roid_id = ezomero.post_roi(conn, image_id=image_id, shapes=rectangles)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we check on our OMERO server we see our predictions:\n",
+    "\n",
+    "![OMERO predictions](./imgs/image8.PNG)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Final thoughts\n",
+    "\n",
+    "That's it! This is quite a lengthy tutorial, so props if you made it all the way here. \n",
+    "\n",
+    "### Used data\n",
+    "\n",
+    "All the data used in this tutorial is [publically available](https://zenodo.org/records/13329153) at the courtesy of Louis Hewitt ([Grapin-Botton Lab](https://www.mpi-cbg.de/research/researchgroups/currentgroups/anne-grapin-botton/research-focus), MPI-CBG, Dresden, Germany). Without their collaboration, you wouldn't be able to look at this tutorial. So, thank you!\n",
```

Comment:
You could perhaps add a link to this section to the very beginning. Whether I could actually reproduce the notebook was something that I asked myself throughout the reading of the blog post.

Reply:
Good point! Done in f735e232764835406e43638887bf718162f29685